portality/templates/includes/header-primary-navigation.html

```diff
@@ -17,8 +17,8 @@
                 {% set urlAttrs = "" %}
             {% endif %}
 
-            <li>
-                <a {{ urlAttrs }} href="{{ url }}" class="dropdown__link">
+            <li role="none">
```

Comment:
We should keep list items as part of the accessibility tree.

Reply:
`role="none"` has been added here in line with the W3 pattern. Why is it not correct?

Reply:
(same comment re: semantics of lists)